<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MAA-5CG0263H4C" failures="0" tests="3" name="TestNg.dependsOnMethod" time="0.022" errors="1" timestamp="2022-12-24T13:58:44 IST" skipped="0">
  <testcase classname="TestNg.dependsOnMethod" name="signup" time="0.016"/>
  <system-out/>
  <testcase classname="TestNg.dependsOnMethod" name="Name" time="0.003">
    <error type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at TestNg.dependsOnMethod.Name(dependsOnMethod.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- Name -->
  <system-out/>
  <testcase classname="TestNg.dependsOnMethod" name="Address" time="0.003"/>
  <system-out/>
</testsuite> <!-- TestNg.dependsOnMethod -->
